src/pytest_plugins/execute/execute.py

```diff
@@ -342,6 +342,9 @@ def __init__(self, *args, **kwargs):
                     kwargs["pre"] = pre
                 elif kwargs["pre"] != pre:
                     raise ValueError("The pre-alloc object was modified by the test.")
+                # Pass gas_benchmark_value if not already set
+                if "gas_benchmark_value" not in kwargs:
+                    kwargs["gas_benchmark_value"] = request.getfixturevalue("gas_benchmark_value")
```

Comment:
We should set the `expected_benchmark_gas_used` to `gas_benchmark_value` here.
Once configured, we do not need to import all the `gas_benchmark_value` parameter in the test wrapper.


```suggestion
                if "expected_benchmark_gas_used" not in kwargs:
                    kwargs["expected_benchmark_gas_used"] = request.getfixturevalue("gas_benchmark_value")
```